open_api_framework/conf/base.py

```diff
@@ -632,6 +632,11 @@ def init_sentry(before_send: Callable | None = None):
 
 # we run the admin site monkeypatch instead.
 TWO_FACTOR_PATCH_ADMIN = False
+# Relying Party name for WebAuthn (hardware tokens)
+TWO_FACTOR_WEBAUTHN_RP_NAME = f"{PROJECT_DIRNAME} - admin"
```

Comment:
This does make the RP name something like `openzaak - admin` instead of `Open Zaak - admin`, do we mind this?